src/test/resources/application.yml

```diff
@@ -119,6 +121,37 @@ nhais:
       +yLvICQmVjHWsNWD3uuT3gEGdeKDIzEnovXUmqhQE7N/YDupTv5hkh8hMKl12XGs
       3WJ4bVSmp3MOQ2EF4zrAl4fXEd4lS3JxKtzbHFbSogAmPFnuP8wIDWTDgfEq
       -----END RSA PRIVATE KEY-----}
+    subCAcert: ${NHAIS_MESH_SUB_CA:-----BEGIN CERTIFICATE-----
+      MIIFXzCCA0egAwIBAgIJALRbCSor9bEbMA0GCSqGSIb3DQEBCwUAMEUxCzAJBgNV
```

Comment:
as above, if this is the opentest intermediate we should omit it since defaults are for fake-mesh

Reply:
this one is from fake-mesh https://github.com/jamespic/fake-mesh/blob/master/src/fake_mesh/ca.cert.pem

Reply:
It's not needed, I've put that so devs would be aware of this var.

Reply:
Removing all subCAcerts for fake-mesh.